float32 gy_x_ang
float32 gy_y_ang
float32 gy_z_ang
